resources/features/VAT.feature

```diff
@@ -0,0 +1,370 @@
+Feature: VAT Calculation
+    The VAT Calculation method will take information from the BI data, VAT Refs and PAYE refs tables calculating various turnovers.
+
+ @HappyPath
+    Scenario Outline: Happy Path - We have valid input data and will calculate the correct result
+        Given the Legal unit input with vat:
+          |BusinessName|			   		PayeRefs|                     VatRefs|       ern|			   id|
+          |  Business 1|			 	 846SZ24053]|			   [848723100000]|1100000001|1000100001234567|
+		  | Business 2A|				    [846T10]|[388206724000, 388206724001]|1100000002|1000100002345678|
+		  | Business 2B|			     [083NU5010]|[388206724002, 388206724003]|1100000002|1000100003456789|
+		  | Business 3A| [070EXB497, 083S7501, 8751]|[741144071000, 741144071001]|1100000003|1000100004567890|
+		  | Business 3B|				  [073S7501]|			   [741144071002]|1100000005|1000100005678901|
+		  | Business 3C|				  [120E7131]|			   [741144071003]|1100000004|1000100006789012|
+		  | Business 6A|				 [120O40781]|			   [142672964000]|1100000006|1000100007890123|
+	      | Business 6B|				 [581MC4245]|			   [142672964001]|1100000006|1000100008901234|
+          | Business 6C|					  [null]|			   [142672964002]|1100000006|1000100009012345|
+	      | Business 6D|					  [null]|			   [524566793000]|1100000006|1000100000123456|
+    	  | Business 7A|				[419YZ08106]|[618020279000, 618020279001]|1100000007|1000100009876543|
+	      | Business 7B|	   [581MS4467, 120T2262]|[618020279002, 644356629000]|1100000008|1000100008765432|
+        And the VAT refs input:
+          |      vatref|turnover|record_type|
+	      |848723100000| 6932069|		   0|
+		  |388206724000| 1925250|		   1|
+		  |388206724001|   15432|		   3|
+	      |388206724002|   80000|		   3|
+	      |388206724003|    1200|		   3|
+		  |741144071000|  293018|	       1|
+		  |741144071001|   78596|	   	   3|
+		  |741144071002|   52154|	       3|
+	      |741144071003|     127|	       3|
+	      |142672964000|26208213|	       1|
+		  |142672964001|   36521|	       3|
+		  |142672964002|   98562|	       3|
+		  |524566793000|   61623|		   0|
+		  |618020279000|  713204|		   1|
+		  |618020279001|    3561|		   3|
+		  |618020279002|    1265|		   3|
+		  |644356629000|    2481|		   0|
+        And the PAYE input:
+          |       ern|paye_empees|paye_jobs|
+		  |1100000001|		25469|	  25484|
+		  |1100000002|		 3721|	   3751|
+		  |1100000003|		29261|	  28699|
+	      |1100000005|			6|	      5|
+	      |1100000004|	   239675|	 239675|
+	      |1100000006|	    11916|	  11946|
+	      |1100000007|	      365|  	365|
+	      |1100000008|	      480|	 	495|
+        When VAT is calculated
+        Then a combination of the PAYE and VAT results tables is produced:
```

Comment:
I'm sure there are good reasons for this - but it feels a little odd.
This test is the "VAT calculation" - and we are saying its output contains PAYE information?
I recognise that the VAT calculation needs employee data as input - but this seems to imply that the VAT calculation method is not a pure VAT calculation method ...  Is the separation of concerns correct here?

Reply:
The VAT variables are the only output of this processing, appending to the enterprise table which includes the PAYE calculated employee values added by the PAYE method.